src/hardware/vga_draw.cpp

```diff
@@ -1614,23 +1614,27 @@ void VGA_SetupDrawing(uint32_t /*val*/)
 	case M_HERC_GFX:
 		vga.draw.blocks=width*2;
 		width*=16;
-		aspect_ratio = (static_cast<float>(width) / static_cast<float>(height)) * (3.0f / 4.0f);
+		assert(width > 0);
+		assert(height > 0);
+		aspect_ratio = static_cast<float>(width * 3) / static_cast<float>(height * 4);
```

Comment:
`assert` already deals with false-positive report of division by zero, changing aspect ratio calculation is not necessary, so I suggest keeping it as it was.